packages/core/src/components/popover/popover.tsx

```diff
@@ -124,6 +124,14 @@ export interface IPopoverProps extends IOverlayableProps, IProps {
     onInteraction?: (nextOpenState: boolean) => void;
 
     /**
+     * Whether the popover should open when its target is focused.
+     * If `true`, target will render with `tabindex="0"` to make it focusable via keyboard navigation.
+     * This prop is only available when `interactionKind` is `HOVER` or `HOVER_TARGET_ONLY`.
```

Comment:
i think i'd prefer to simply mix this behavior into `HOVER` and `HOVER_TARGET_ONLY`, such that hover and focus interactions are treated the same (which we do in the CSS for many components), rather than adding a new prop that only applies some of the time.